### DISCUSSION TITLE
Deployment not working in vercel, 404 NOT_FOUND

### DISCUSSION THREAD

**Discussion by kiongosss** [Resume-Genie/resume-genie#17] [Q&A] [answered]
# Bug Report

## Description

404: NOT_FOUND
Code: NOT_FOUND
ID: cpt1::sbbz4-1687361083841-76166a29449c

After deploying it

URL: https://resume-genie-main-k4ee.vercel.app/

**Comment by Gautam25Raj** [comment, marked as answer]
@kiongosssIt appears that you are encountering difficulties deploying the project on `Vercel` due to the separation of the frontend and backend into separate folders. To successfully deploy the project on `Vercel`, please follow the recommended deployment steps:

- Make sure you are in the root directory of the project.
- Open the `"client"` folder, which contains the frontend code.
- Configure the `Vercel` deployment settings to use the `"client"` folder as the root directory for the deployment.
- Ensure that all necessary frontend dependencies are installed within the `"client"` folder.
- Once the deployment settings are configured correctly, initiate the deployment process on `Vercel`.

By deploying the project using the "client" folder as the root directory, `Vercel` will recognize the frontend code and successfully host and serve your application. Please note that if your project has a separate backend server, it should be deployed separately using the appropriate deployment process for the backend technology.

## Steps to Deploy
<p align="center">
  <img src="https://github.com/Gautam25Raj/resume-genie/assets/63155224/5b035ca5-b7e6-48d0-9e5c-74c14f2c8327" width="500px;" alt=""/>
</p>
<p align="center">
  <img src="https://github.com/Gautam25Raj/resume-genie/assets/63155224/001c5fb1-fbce-4a02-9e82-c1c4c460ec50" width="500px;" alt=""/>
</p>
<p align="center">
  <img src="https://github.com/Gautam25Raj/resume-genie/assets/63155224/64238476-0fb5-4570-8a72-e74d3e1cad1e" width="500px;" alt=""/>
</p>
<p align="center">
  <img src="https://github.com/Gautam25Raj/resume-genie/assets/63155224/c8da3562-8274-4b7b-8c09-f4b5b428f436" width="500px;" alt=""/>
</p>

**Reply by kiongosss** [reply]
Worked like magic

**Reply by Gautam25Raj** [reply]
Good to know it helped.
